tests/test_gather_stock_time_series.py

```diff
@@ -0,0 +1,88 @@
+import pyntrinio
+from datetime import datetime
+
+# helper data
+api_key = 'OjhlMjhjNTBmY2IyMWJiMWE0MTExYjQwNWZmZTVkZWM1'
+ticker = 'AAPL'
+start_date = '2005-03-01' 
+end_date = '2020-01-15' 
+
+# test that return type is dictionary by default
+def test_default_output_type():
+    """
+    Test that return type is dictionary by default
+    """
+    assert type(gather_stock_time_series(api_key, ticker)) == 'dict'
```

Comment:
I'm wondering whether doing the tests without specifying dates could lead to traffic control from the API server?  I remember we mentioned something like `1 min` restriction.   
And also the sake of efficiency, we might want to set a small time range so that the codes could be executed more quickly.

Reply:
addressed! also kept the issue open in hopes to address this issue before submission